controllers/update_status.go

```diff
@@ -82,7 +82,16 @@ func (updateStatus) reconcile(r *FoundationDBClusterReconciler, context ctx.Cont
 
 		databaseStatus, err = adminClient.GetStatus()
 		if err != nil {
-			return &requeue{curError: err}
+			if cluster.Status.Configured {
+				return &requeue{curError: err}
+			}
+			databaseStatus = &fdbtypes.FoundationDBStatus{
```

Comment:
I think that should be fine to assume the DB is not configured. One think that can happen, but shouldn't cause any issues, is that the DB get's configured again with the same settings.

Do you think it makes sense to move the creation of the status into a method to reduce the duplication?

Reply:
I think it's only a small amount of code, so it's fine to have a little duplication.